packages/components/src/List/ItemTitle/ItemTitle.scss

```diff
@@ -23,6 +23,8 @@ $tc-list-item-title-icon-size: $svg-lg-size !default;
 		overflow: hidden;
 		text-overflow: ellipsis;
 		white-space: nowrap;
+		text-transform: none;
```

Comment:
except the rule inherit from bootstrap set all buttons with a text-transform to uppercase